try_install_all.py

```diff
@@ -24,7 +24,8 @@
     ]
 SCRIPT_LIST = SCRIPT_LIST()
 TEST_ENGINES = {}
-IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet"]
+IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet","NPN", "mammsupertree", "eBirdOD"]
```

Comment:
I don't think we should ignore NPN or eBirdOD since these are legitimately erroring at the moment. We should fix those errors via #562 instead.